test/game_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'simplecov'
+SimpleCov.start
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/game'
+
+class GameTest < Minitest::Test
+
```

Comment:
I'm a fan of setup methods to clean up the test file. I would add the game instance in a setup method here.